packages/components/src/components/base/CreatedModified.tsx

```diff
@@ -32,9 +34,36 @@ interface IRowConfig {
 interface CreatedModifiedProps {
     className?: string
     row: Map<string, any>
+    userServerDate?: boolean
 }
 
-export class CreatedModified extends React.Component<CreatedModifiedProps, any> {
+interface State {
+    serverDate: string
+}
+
+export class CreatedModified extends React.Component<CreatedModifiedProps, State> {
+
+    static defaultProps = {
+        userServerDate: true
+    };
+
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            // undefined will be used as the loading state until Query.getServerDate is back, null means to use current date from JS
```

Comment:
Would it be better to use a boolean loadingDate field in the state?  I can see this works, but the distinction between null and undefined seems a little sketchy.

Reply:
Yes, easy enough to add a loading state variable. Thanks.